2-virtual-machines-containers/2.1-golden-images.md

```diff
@@ -5,14 +5,35 @@
 ![](img2/linux_light.svg ':size=100x100 :class=light-mode-icon')
 ![](img2/linux_dark.svg ':size=100x100 :class=dark-mode-icon')
 
+?> If you have an m1 mac checkout virtualbox beta build for arm64. ( Note that as of 23/02/2023 the virtualbox beta build for arm64 did not work)
+
+?> assuming you have an m1 mac & the virtualbox beta is still not operational, the only m1 compatible section is the compatibility section.
+
+### M1 Compatibility Excersize
```

Comment:
Is there a reason you kept the VirtualBox section? I have not gone through the whole Bootcamp yet but I feel like we never go back to virtual box and we might just be able to change UTM.

Reasons to keep both that I can think of:
1) VirtualBox is cross platform UTM is not

Reasons to get rid of VirtualBox:
1) In the Introduction we already specify this Bootcamp is written for Mac users so the reason to keep it might not matter
2) Now we need to maintain two sets of instructions.

I searched through the bootcamp and only saw virtualbox being referenced in chapter 2. With that I feel like we should just switch to UTM. I am assuming that UTM works well across intel and apple silicon macs. And I see this section really as a way to just introduce the concept of virtual machines and get some hands on with it.

Reply:
It feels like you went into too much detail in this section. The previous instructions were
>Using [VirtualBox](https://www.virtualbox.org/wiki/Downloads), create a new virtual machine and install CentOS using the ISO you downloaded.

I am inclined to say this should just say something like
>Using [UTM](https://mac.getutm.app/), create a new virtual machine and install CentOS using the ISO you downloaded.

I think this holds their hand less, will be easier to maintain down the road (UIs change, and specific instructions are prone to going stale). Additionally this sends the user to a page about UTM where they can read about it. Personally this was software I had never heard of so when asked to just `brew` install UTM I was like "what is it and why am I installing it"

Reply:
After discussing this further with Evan I am convinced we should keep both the VirtualBox and UTM sections in. UTM might get cleaned up down the road when VirtualBox has a working build for arm processors. And I see this section as a way to introduce Apprentices to virtualization and we move past it pretty quickly. 

I do think we could simplify the step-by-step instructions to be more inline with the existing VirtualBox instructions but this is not a blocker.

Reply:
this should be fixed in new commit